src/s3/getobject.cpp

```diff
@@ -112,14 +171,18 @@ asio::awaitable<void> irods::s3::actions::handle_getobject(
                 }
                 catch (boost::system::system_error& e) {
                     if (e.code() != beast::http::error::need_buffer) {
-                        std::cout << "Not a good error!" << std::endl;
+                        std::cout << "Not a good error!" << e.code() << std::endl;
```

Comment:
I think `system_error` contains a message as well. Let's print that out too.

Also, let's change the `Not a good error!` to something that helps the admin better or consider removing it altogether.

Reply:
> I think `system_error` contains a message as well. Let's print that out too.
> 
> Also, let's change the `Not a good error!` to something that helps the admin better or consider removing it altogether.

Yep, I didn't write that.  I'll change the message and add e.code().message().

Reply:
I changed the message and added e.code().message() to the output.